datafusion/sqllogictest/test_files/join.slt

```diff
@@ -1152,7 +1152,7 @@ logical_plan
 01)Projection: t1.v0, t1.v1, t5.v2, t5.v3, t5.v4, t0.v0, t0.v1
 02)--Inner Join: CAST(t1.v0 AS Float64) = t0.v1 Filter: t0.v1 + CAST(t5.v0 AS Float64) > Float64(0)
 03)----Projection: t1.v0, t1.v1, t5.v0, t5.v2, t5.v3, t5.v4
-04)------Inner Join: Using t1.v0 = t5.v0, t1.v1 = t5.v1
+04)------Inner Join: t1.v0 = t5.v0, t1.v1 = t5.v1
```

Comment:
This basically seems like it is the same plan to me (just with a different syntax 🤔 )

Reply:
Yeah it's just the rule now "rewrites" it, before it didn't because of the filter in the join above it.